src/components/auth/signup/signup-modal.tsx

```
@@ -0,0 +1,69 @@
+            <Input placeholder="비밀번호" label="비밀번호" size="large" fullWidth required />
+            <Input placeholder="비밀번호 확인" label="비밀번호 확인" size="large" fullWidth required />
+            <div className={s.checkbox_container}>
+                <Checkbox 
+                    className={s.checkbox_text}
+                    checked={isTermsAgreed}
+                    onCheckedChange={setIsTermsAgreed}
+                    label={
+                        <span>
+                            눌러서&nbsp;
+                            <a 
+                                href="/terms" 
```

ESLint

[error] Replace `································href="/terms"·` with `··················href="/terms"`

(prettier/prettier)